WebContent/read-dataset.jsp

```diff
@@ -23,10 +23,13 @@
 <div class="logo" style="text-align:center">Surf</div>
     <div class="user-input">
 
-        <form method="get" action="select-op">
+        <%-- <form method="get" action="select-op"> --%>
+        <form method="post" action="columnUpdate">
         <%
             // Set up dataset in the session
+            
             Dataset dataset = MergeServlet.loadDataset(request);
+            Config.refreshCols((String) session.getAttribute("datasetKey"));
```

Comment:
I'm refreshing the columns in read dataset and sending the selected colsToShow to columnUpdateServlet through a post request. However, once the user has unselected cols and goes back and tries to select them, the cols are not showing up in the final table despite refreshing.

If however, the user reloads the dataset, it seems to work.

Planning to fix this issue soon.

Reply:
fixed.